metrics_service/settings.py

```diff
@@ -0,0 +1,361 @@
+# WARNING: DO NOT EDIT THIS FILE, IT IS MANAGED BY ANSIBLE-SERVICES-FRAMEWORK.
+"""# The main settings for metrics_service.
+
+## On this module
+
+1. Default Settings (regular django settings)
+2. Dynaconf instrumentation that loads external settings including
+[DAB](https://github.com/ansible/django-ansible-base) defaults
+
+## Loading order
+
+1. Project defaults as defined in the first section of this file.
+2. DAB defaults.
+3. Application defaults.
+   - Top level apps/settings.py
+   - Each `app/settings.py` in `apps` directory.
+4. DAB conditional settings.
+   - Loaded based on what is on current INSTALLED_APPS, MIDDLEWARES etc
+5. Overrides from standard paths.
+   - /etc/ansible-automation-platform/metrics_service/
+6. Overrides from environment variables prefixed with {{project_name | upper }}_.
+7. Post loading hooks registered on any Python file loaded before.
+
+## Overrides
+
+> [!NOTE]
+> To override defaults define variables on each `apps`/app_name/settings.py.
+> To override settings on environments use the override path `/etc/...`
+> or set environment variables prefixed with `{{project_name | upper }}_`.
+> The environment variables can also be used on development environments
+> by adding a `.env` file to the root of the project or creating a
+> `settings.local.py`
+
+## Inspecting settings and troubleshooting
+
+To examine the loading history of the settings loading mechanism.
+
+```bash
+# Export the Django Settings Module Variable
+export DJANGO_SETTINGS_MODULE=metrics_service.settings
+
+# The whole history
+uv run dynaconf inspect -m debug -f yaml
+
+# Inspect a variable
+uv run dynaconf inspect -k VARIABLE
+```
+
+## Default variables
+"""
+
+import os
+import sys
+from importlib import import_module
+from pathlib import Path
+
+from ansible_base.lib.dynamic_config import (
+    export,
+    factory,
+    load_dab_settings,
+    load_envvars,
+    load_standard_settings_files,
+)
+from dynaconf.loaders import execute_instance_hooks
+
+BASE_DIR = Path(__file__).resolve().parent.parent
+"""Build paths inside the project like this: BASE_DIR / 'subdir'"""
+
+SECRET_KEY = "django-insecure-k^a&fnx3ulh*d2nl%q680o+xkr^5o+c$5=lzo7vd-7=#qmadg("  # noqa
+"""SECURITY WARNING: keep the secret key used in production secret!"""
+
+DEBUG = False
+"""SECURITY WARNING: don't run with debug turned on in production!"""
+
+ALLOWED_HOSTS = []
```

Comment:
```suggestion
ALLOWED_HOSTS = ['*']
```

and probably remove it from apps/defaults.py